src/main/java/wooteco/subway/line/LineController.java

```diff
@@ -0,0 +1,62 @@
+package wooteco.subway.line;
+
+import org.springframework.dao.DataAccessException;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.net.URI;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@RestController
+public class LineController {
+
+    private final LineDao lineDao;
+
+    public LineController(LineDao lineDao) {
+        this.lineDao = lineDao;
+    }
+
+    @PostMapping("/lines")
+    public ResponseEntity<LineResponse> createLine(@RequestBody LineRequest lineRequest) {
+        Line line = new Line(lineRequest.getName(), lineRequest.getColor());
+        Long id = lineDao.save(line);
+
+        Line newLine = new Line(id, lineRequest.getName(), lineRequest.getColor());
+        LineResponse lineResponse = new LineResponse(newLine.getId(), newLine.getName(), newLine.getColor());
```

Comment:
```suggestion
        LineResponse lineResponse = new LineResponse(line.getId(), lineRequest.getName(), lineRequest.getColor());
```
와 같이 작성한다면 26라인은 제거해도 괜찮을 것 같네요.